options/locale/locale_en-US.ini

```
@@ -3230,3 +3234,58 @@ deletion = Remove secret
 deletion.description = Removing a secret will revoke its access to repositories. Continue?
 deletion.success = The secret has been removed.
 deletion.failed = Failed to remove secret.
+
+[actions]
+actions = Actions
+
+unit.desc = Manage actions
+
+status.unknown = "Unknown"
+status.waiting = "Waiting"
+status.running = "Running"
+status.success = "Success"
+status.failure = "Failure"
+status.cancelled = "Cancelled"
+status.skipped = "Skipped"
+status.blocked = "Blocked"
+
+runners = Runners
+runners.runner_manage_panel = Runners Management
+runners.new = Create new Runner
+runners.new_notice = How to start a runner
+runners.status = Status
+runners.id = ID
+runners.owner_type = Type
+runners.description = Description
+runners.labels = Labels
+runners.last_online = Last Online Time
+runners.agent_labels = Agent Labels
+runners.custom_labels = Custom Labels
+runners.custom_labels_helper = Custom labels are labels that are added manually by an administrator. A comma separates labels, whitespace at the start and end of each label are ignored.
```

```suggestion
runners.custom_labels_helper = Custom labels are labels that are added manually by an administrator. A comma separates labels, whitespace at the start and end of each label is ignored.
```
(Just noticed that I made a mistake there - it read a little weird.)